crates/control/src/motion/arms_up_squat.rs

```diff
@@ -14,22 +19,40 @@ pub struct CreationContext {
 pub struct CycleContext {
     pub motion_safe_exits: PersistentState<MotionSafeExits, "motion_safe_exits">,
 
-    pub motion_selection: RequiredInput<Option<MotionSelection>, "motion_selection?">,
+    pub motion_selection: Input<MotionSelection, "motion_selection">,
```

Comment:
Is it possible to have a non-optional motion selection? Do the previous nodes always output something?

Reply:
Yes. #worksForMe I think a walking robot proves it

Reply:
And additionally it should! We should never be in a state in which we cannot know which joint angles to send. There should always be at least an unstiff